src/LessonOOPstarter/Solved/BankAccount/BankAccount.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace BankAccount
+{
+    public class BankAccount
```

Comment:
Sa nu uitam sa le spunem si despre **this**

Reply:
atunci o sa adaug this, ca sa fie evident si in exemplu